src/app/recording/Recording.js

```diff
@@ -316,7 +316,7 @@ function Recording () {
   }
 
   async function onFeatureCreated (feature) {
-    const createdFeature = await postSegment(feature)
+    const createdFeature = await postSegment({ ...feature, properties: { subsegments: [] } })
```

Comment:
This is just a placeholder, right?

Reply:
ah okay, I see

Reply:
Yep! probably needs to be replaced at some point.